lib/sufia/engine.rb

```diff
@@ -72,6 +73,7 @@ class Engine < ::Rails::Engine
     config.temp_file_base = nil
     config.redis_namespace = "sufia"
     config.fits_path = "fits.sh"
+    config.libreoffice_path = "/opt/local/libreoffice_path/bin/soffice"
```

Comment:
`soffice` is the present default: https://github.com/projecthydra/hydra-derivatives/blob/252cabb1278f703d20805d813bdc5c23ca056a8c/lib/hydra/derivatives/config.rb#L13